library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -1971,13 +1971,15 @@ void libspdm_register_device_buffer_func(
     context->acquire_receiver_buffer = acquire_receiver_buffer;
     context->release_receiver_buffer = release_receiver_buffer;
 
-    LIBSPDM_ASSERT (sender_buffer_size >= LIBSPDM_TRANSPORT_ADDITIONAL_SIZE);
-    sender_buffer_size -= LIBSPDM_TRANSPORT_ADDITIONAL_SIZE;
+    LIBSPDM_ASSERT (sender_buffer_size >=
```

Comment:
`transport_additional_size` and `max_spdm_msg_size` should be set in `libspdm_register_device_buffer_func`.

Reply:
agree. Updated.